frontend/app/src/connection/DefaultStreamlitEndpoints.ts

```diff
@@ -81,6 +95,10 @@ export class DefaultStreamlitEndpoints implements StreamlitEndpoints {
    * the URL alone.
```

Comment:
nit: I think the JSDocs here have to be updated to inform about the newly added logic

Reply:
Done!